src/core_ocean/shared/mpas_ocn_gm.F

```diff
@@ -681,29 +684,19 @@ subroutine ocn_GM_compute_Bolus_velocity(statePool, diagnosticsPool, &
                        + 1.0e-18_RKIND )
                  RiTopOfEdge = BruntVaisalaFreqTopEdge * shearEdgeInv
                  sigma = max(abs(fEdge(iEdge)),sqrt(2.0_RKIND*betaEdge(iEdge)* &
-                             cGMphaseSpeed(iEdge))) / sqrt(RiTopOfEdge + config_gm_EG_riMin)
+                             cGMphaseSpeed(iEdge))) / sqrt(RiTopOfEdge + config_GM_EG_riMin)
                  L_rhines = sigma / (1.0E-18_RKIND + betaEdge(iEdge))
-                 Length = min(config_gm_EG_Rossby_factor*Lr,config_gm_EG_Rhines_factor*L_rhines)
+                 Length = min(config_GM_EG_Rossby_factor*Lr,config_GM_EG_Rhines_factor*L_rhines)
                  !For compatibility with other schemes, we normalize by the 2-D kappa field, which will
                  !be removed below in the computation of the stream function
-                 gmKappaScaling(k,iEdge) = max(config_GM_Visbeck_min_kappa, min(config_gm_EG_kappa_factor* &
-                         Length**2.0*sigma, config_GM_Visbeck_max_kappa)) / gmBolusKappa(iEdge)
+                 gmKappaScaling(k,iEdge) = max(config_GM_spatially_variable_min_kappa, min(config_GM_EG_kappa_factor* &
+                         Length**2.0*sigma, config_GM_spatially_variable_max_kappa)) / gmBolusKappa(iEdge)
               enddo
            enddo
            !$omp end do
            !$omp end parallel
          endif
 
-         if(config_Redi_set_RediKappa_to_GMKappa) then
```

Comment:
We decided to remove this flag, so I removed this block completely. @vanroekel please double check this deletion.

Reply:
looks good, thanks!